framework/include/base/FEProblem.h

```diff
@@ -832,6 +832,11 @@ class FEProblem :
   Adaptivity & adaptivity() { return _adaptivity; }
   virtual void initialAdaptMesh();
   virtual void adaptMesh();
+  unsigned int _cycles_completed;
+  /**
+   * @return The number of adaptivity cycles completed.
+   */
+  unsigned int getNumCycles() { return _cycles_completed; }
```

Comment:
Can you make this `getNumCyclesCompleted()`?  The way it's worded now you might think that it's going to give you back the number of cycles _prescribed_.